pkg/internal/sqlsmith/schema.go

```diff
@@ -248,6 +272,35 @@ func extractIndexes(
 					Direction: dir,
 				})
 			}
+			row, err := db.Query(fmt.Sprintf(`
```

Comment:
if you expect exactly one row you can use `db.QueryRow('query').Scan(&blah)`

Reply:
Oh, nice, thanks!